src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Debug/ConsoleDebugLaunchProviderTests.cs

```diff
@@ -391,25 +391,7 @@ public void GetExeAndArgumentsWithEmptyArgs()
             var targets = await debugger.QueryDebugTargetsAsync(0, activeProfile);
 
             Assert.Single(targets);
-            Assert.Equal(@"C:\library.dll", targets[0].Executable);
-        }
-
-        [Fact]
-        public async Task QueryDebugTargetsAsync_WhenLibraryWithoutRunCommand_DoesNotManipulateTargetPath()
```

Comment:
You should make this test pass, as above this is testing Test Explorer scenarios.